docs/layers/network/dns-setup.mdx

```diff
@@ -12,9 +12,9 @@ import StepNumber from '@site/src/components/StepNumber';
 
 ## Requirements
 
-Before deploying DNS, first purchase your chosen vanity and service domains in the `core-dns` account or in your chosen registrar. Refer back to the [Decide on Vanity (Branded) Domain](layers/network/design-decisions/decide-on-vanity-branded-domain/) and [Decide on Service Discovery Domain](/layers/network/design-decisions/decide-on-service-discovery-domain/) design decisions for more information.
+Before deploying DNS, first purchase your chosen vanity and service domains in the `core-dns` account or in your chosen registrar. Refer back to the [Decide on Vanity (Branded) Domain](/layers/network/design-decisions/decide-on-vanity-branded-domain/) and [Decide on Service Discovery Domain](/layers/network/design-decisions/decide-on-service-discovery-domain/) design decisions for more information.
```

Comment:
It's not required to purchase them in the `core-dns` account.

Reply:
> in the `core-dns` account or in your chosen registrar.

Or in your chosen registrar

Reply:
Do we have instructions for delegating NS from an external registrar to a route53 zone? E.g. first create the zone in route53, then delegate the nameservers at the registrar

Reply:
we have a bit in this same page
![CleanShot 2024-08-09 at 11 28 54@2x](https://github.com/user-attachments/assets/f906cf1a-b964-4e38-a391-db0a0a0d282e)